preferences/centos/10_stream/kustomization.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../base
+
+components:
+  - ./metadata
+  - ./requirements
+  - ../../components/rng
+  - ../../components/disk-dedicatediothread
```

Comment:
@asamalik should the CentOS Stream 10 cloud image but with BIOS, like CentOS Stream 9, or with UEFI and secure boot like RHEL ?

Reply:
Right now it is a straight copy from CentOS Stream 9.

Reply:
@tdawson should the CentOS Stream 10 cloud image but with BIOS, like CentOS Stream 9, or with UEFI and secure boot like RHEL ?